src/main/resources/templates/admin/defaultVoters.html

```diff
@@ -12,14 +12,21 @@
 <h2>Default Voters</h2>
 
 <p>
-    Only users with Terraformation email addresses may be default voters. Default voters are
-    assigned as voters to participant projects.
+    Default voters are assigned as voters to participant projects. If the checkbox under a project
+    is checked, the add/remove will propagate to that project current phase of voting. By default,
+    projects with incomplete votes will inherit the voter change.
 </p>
 
 <table>
     <thead>
     <tr>
         <th>Email</th>
+        <th>Action</th>
+        <th>Projects:</th>
+        <th th:each="project : ${projects}"
```

Comment:
Would it make sense to use a `<select multiple>` for this instead of a one-column-per-project approach? I'm concerned about what this UI will look like when there are a couple dozen projects.

Alternately, maybe rotate this 90 degrees so the voters are the columns and the projects are the rows.